app/views/application_submissions/past_applications.haml

```diff
@@ -23,6 +24,11 @@
             = format_date_time record.interview.scheduled
         %td
           = record.staff_note
+        %td 
+          - if record.rejection_message.present?
+            = record.rejection_message
+          - else
+            none
```

Comment:
This is matter of personal preference, but I think a blank cell would be preferable to 'none'

Reply:
In fact, maybe you could dry this up and have just `= record.rejection_message`, depending on whether its stored as nil or '' and how `= nil` behaves with HAML

Reply:
HAML doesn't care about nil, see staff_note display above.